libs/sdk-ui-tests-e2e/cypress/tools/filterBar.ts

```diff
@@ -486,6 +486,58 @@ export class AttributeFilter {
         return this;
     }
 
+    openAddLimitDashboardFilter() {
+        cy.get(".s-add").click();
+        cy.get(".s-add-limit-dashboard_filter").click();
+    }
+
+    closeAddLimitDashboardFilter() {
+        cy.get(".configuration-panel-header-title.clickable").click();
+        return this;
+    }
+    isSpecificDateFilterVisible(parentFilterName: string, visible = true): this {
+        this.openAddLimitDashboardFilter();
+        cy.get(".s-dashboard-filter-" + parentFilterName).should(($element) => {
+            if (visible) {
+                // eslint-disable-next-line jest/valid-expect
+                expect($element).to.have.class("is-disable");
+            } else {
+                // eslint-disable-next-line jest/valid-expect
+                expect($element).not.to.have.class("is-disable");
+            }
+        });
+        this.closeAddLimitDashboardFilter();
+        return this;
+    }
+    isCommonDateFilterVisible(parentFilterName: string, visible = true): this {
```

Comment:
Please add some information for this method. It should be the same method "configureLimitingMetricDependency"